protocol/src/codec/crosschain.rs

```diff
@@ -82,6 +107,72 @@ impl Decodable for HashWithDirection {
     }
 }
 
+#[derive(Clone)]
+pub struct IbcWrapper<T: Clone>(pub T);
```

Comment:
```suggestion
pub struct IbcWrapper<T: Message>(pub T);
```

Reply:
Not all the `ibc item` implements the prost::Message